packages/govern-console/src/components/ActionBuilder/Screens/AbiForm.tsx

```diff
@@ -12,6 +12,7 @@ import {
   SPACING,
   Info,
 } from '@aragon/ui';
+import Switch from '@material-ui/core/Switch';
```

Comment:
I didn't know how to use `components/Switchs/BlueSwitch.tsx` and didn't find an example.